crates/tombi-ast/src/impls/comment.rs

```diff
@@ -33,7 +34,7 @@ impl Comment {
                 }
 
                 Some((
-                    url::Url::from_file_path(&schema_file_path).map_err(|_| url_str.to_string()),
+                    url_from_file_path(&schema_file_path).map_err(|_| url_str.to_string()),
```

Comment:
Let me think a bit about how to handle Url.

I'm considering moving to [tower-lsp-server](https://github.com/tower-lsp-community/tower-lsp-server), but the current latest lsp-types::Url is switching to fluent_uri crates.


The file_path -> url problem is happening [there](https://github.com/tower-lsp-community/tower-lsp-server/issues/34) too, and I'm considering wrapping it with `tombi-url` crates.

Reply:
We will hold off on `tombi-url` and give priority to wasm support.

It may not be worth the cost to have tombi's own Url implementation in `tombi-url`.

Reply:
Then, would it be acceptable for now to create a simple `tombi-url` crate containing only the contents of `tombi-wasm-compat::url.rs` from this PR? (In any case, we'll need to introduce a compatibility layer somewhere.)

Reply:
Yes, let's create `tombi-url` as a compatibility layer.

Please create another PR 🙇